### PR TITLE
Plandev zoningcom conversion

### PR DESCRIPTION
**Pull request by bechatman**

### CODE REVIEW

**Comment by bechatman** [comment]
That worked great. I have a feeling this is going to come up a lot as I move files in, though, so should I be running rake on the repo locally before starting my test builds? Also, for new files, do I just copy the the tasks and change the file name?

**Comment by gregoryjscott** [comment]
Changing the filename in the Rakefile manually will work but we could come up with something better. Want to create a new issue to build something more generic?

**Comment by bechatman** [comment]
Just created it as issue #59. I still have a few more tasks and csv files to add on this before I merge it back to master, so do I just need to type `rake` in to terminal to run that cleanup?

File: Rakefile [hunk 1]
```diff
@@ -12,5 +12,13 @@ namespace :ci do
 
 end
 
+desc 'Remove unrepresentable characters'
+task :scrub_file do
+  filename = File.join('_data', 'zoning-cases.csv')
```
Comment (gregoryjscott) [inline Rakefile]: As of right now you'll have to manually change this line to contain the correct filename. Then you can run `bundle exec rake scrub_file` to scrum that file.

Comment (gregoryjscott) [inline Rakefile]: BTW, you can run `bundle exec rake -T` to see all the available rake commands.

**Pull request merged**